test/e2e/user/expected/requestId.yaml

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
```

Comment:
Where could verify `requestId` in this e2e test? I think I miss it

Reply:
covered. https://github.com/apache/skywalking-nginx-lua/blob/19eaae6947e3552bb2fdefa1d79aa6dcd4f8559c/test/e2e/user/config.yaml#L23-L29

Reply:
I mean, we just would verify codes work, but can see whether request ID is used, right?